docs/endpoints/constructors.md

```diff
@@ -0,0 +1,178 @@
+[← Documentation Home](/docs/README.md)
+# Constructors
+
+Gets list of constructors 
+
+**URL** : `/ergast/f1/constructors`
```

Comment:
Small nitpick, all urls need to end with a `/` or `.json` as the framework automatically redirects urls without a `/` or `.` to one with one.

So `/ergast/f1/constructors/` or `/ergast/f1/constructors.json`